dbt_odis/models/bronze/population_menages_2021.sql

```diff
@@ -1,9 +1,13 @@
 {{ config(
     tags = ['bronze', 'population'],
-    alias='vw_population_menages_2021'
+    alias='vw_population_nb_menages'
     )
 }}
 
-select 
-    {{ dbt_utils.star(from=source('bronze', 'population_menages_td_men1_2021')) }}
-from {{ source('bronze', 'population_menages_td_men1_2021') }}
\ No newline at end of file
+with population_nb_menages as 
+(
+    select {{ dbt_utils.star(from=source('bronze', 'population_menages_td_men1_2021')) }}
+    from {{ source('bronze', 'population_menages_td_men1_2021') }}
```

Comment:
```suggestion
, '2021' as "year"
    from {{ source('bronze', 'population_menages_td_men1_2021') }}
```

On peut ajouter l'année dès la première couche pour avoir l'info au plus tôt.
On peut même utiliser une constante qui serait définie plus haut, du style
```
{% set local_year = '2021' %}
```

et 


```
'{{ local_year }}' as "year"
```